src/main/java/org/sagebionetworks/web/client/widget/modal/Dialog.java

```diff
@@ -32,18 +39,17 @@
 	Button primaryButton;
 	@UiField
 	Button defaultButton;
-	
-	boolean autoHide;
-	
 	@UiField
 	Modal modal;
+	
+	boolean autoHide;
+	Widget widget;
 
 	/**
 	 * Create a new Modal dialog.
 	 */
-	@Inject
-	public Dialog(DialogUiBinder binder) {		
-		initWidget(binder.createAndBindUi(this));
+	public Dialog() {		
+		widget = uiBinder.createAndBindUi(this);
```

Comment:
Move GWT.create(DialogUiBinder.class) to constructor (instead of a one-time view reference)?